command/pr.go

```diff
@@ -72,6 +73,12 @@ var prCloseCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(1),
 	RunE:  prClose,
 }
+var prReopenCmd = &cobra.Command{
+	Use:   "reopen [{<number> | <url>}]",
```

Comment:
The square brackets make this argument seem as if it's optional, but in the implementation it's required